### PR TITLE
Functions: small fixes in scripts

### PR DESCRIPTION
**Pull request by pinebit**

### CODE REVIEW

File: core/scripts/functions/src/sample_config.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "MaxRequestBatchSize": 10,
     "DefaultAggregationMethod": 0,
     "UniqueReports": true,
+    "MaxReportTotalCallbackGas": 500000,
```
Comment (bolekk) [inline core/scripts/functions/src/sample_config.json:9]: We can set it to a much larger value, how about 2M?

File: core/scripts/functions/templates/oracle.toml [hunk 2]
```diff
@@ -11,17 +11,25 @@ relay = "evm"
...
 
 [relayConfig]
 chainID = {{chain_id}}
 
 [pluginConfig]
+contractUpdateCheckFrequencySec = 30
+contractVersion = 1
+donID = "{{don_id}}"
+enableRequestSignatureCheck = false
+listenerEventHandlerTimeoutSec = 120
```
Comment (bolekk) [inline core/scripts/functions/templates/oracle.toml:26]: Let's extend this to 180, as we support flags extending execution time now.

**Review by bolekk — CHANGES_REQUESTED** [review]
Thanks! Can we split out the change to LogPollerWrapper and re-title this PR to only fixes in scripts?

File: core/scripts/functions/templates/oracle.toml [hunk 3]
```diff
@@ -11,17 +11,25 @@ relay = "evm"
...
 """
 
 [relayConfig]
 chainID = {{chain_id}}
 
 [pluginConfig]
+contractUpdateCheckFrequencySec = 30
+contractVersion = 1
+donID = "{{don_id}}"
+enableRequestSignatureCheck = false
```
[the remark this answers is not in the document]
Comment (pinebit) [reply core/scripts/functions/templates/oracle.toml:25]: This is to be explicit that we reviewed this setting and set it to false intentionally.

**Review by bolekk — APPROVED** [review]

**Comment by bolekk** [comment]
Changed title to "small fixes in scripts" I hope that's OK.

File: core/scripts/functions/templates/oracle.toml (hunk 3 above)
[the remark this answers is not in the document]
Comment (KuphJr) [reply core/scripts/functions/templates/oracle.toml:25]: Once audits are over, we could probably just remove this field and the logic supporting this feature entirely given we won't be requiring request signatures.

File: core/scripts/functions/templates/oracle.toml [hunk 4]
```diff
@@ -11,17 +11,25 @@ relay = "evm"
...
 transmitterID = "{{node_eth_address}}"
 observationSource  = """
-	run_computation    [type="bridge" name="ea_bridge" requestData="{\\"note\\": \\"observationSource is unused but the bridge is required\\"}"]
-	run_computation
+    run_computation    [type="bridge" name="ea_bridge" requestData="{\\"note\\": \\"observationSource is unused but the bridge is required\\"}"]
+    run_computation
 """
 
 [relayConfig]
 chainID = {{chain_id}}
```
Comment (KuphJr) [inline core/scripts/functions/templates/oracle.toml:19]: Was this bracket deletion intentional?

Comment (bolekk) [reply core/scripts/functions/templates/oracle.toml:19]: It's not a deletion IIUC, just weird coloring.

**Pull request merged**